components/nexus-elasticsearch/pom.xml

```diff
@@ -37,6 +37,12 @@
       <groupId>org.apache.servicemix.bundles</groupId>
       <artifactId>org.apache.servicemix.bundles.elasticsearch</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.vividsolutions</groupId>
+      <artifactId>jts</artifactId>
+      <version>1.13</version>
```

Comment:
Will do use build support.
is needed by elastic search and is not included in service mix bundle